tensorboard/plugins/mesh/summary_v2.py

```diff
@@ -137,39 +125,33 @@ def mesh(name, vertices, faces=None, colors=None, display_name=None,
   with summary_scope(name, 'mesh_summary', values=tensors):
     for tensor in tensors:
       all_success = all_success and _write_summary(
-        name, display_name, description, tensor.data, tensor.content_type,
+        name, description, tensor.data, tensor.content_type,
         components, json_config, step)
 
   return all_success
 
 
-def mesh_pb(name,
-       vertices,
-       faces=None,
-       colors=None,
-       display_name=None,
-       description=None,
-       config_dict=None):
+def mesh_pb(name, vertices, faces=None, colors=None, description=None,
+    config_dict=None):
   """Create a mesh summary to save in pb format.
 
   Args:
-    name: A name for this summary operation.
+    name: Uniquely identifiable name of the summary op. Could be replaced by
```

Comment:
For the `_pb` version we've been calling this `tag` instead, with the following standard text: `tag: String tag for the summary.`  Mind copying that language for consistency?

https://github.com/tensorflow/tensorboard/blob/dacf300259f850740424a1c228d0a693cffda548/tensorboard/plugins/scalar/summary_v2.py#L68-L72
https://github.com/tensorflow/tensorboard/blob/dacf300259f850740424a1c228d0a693cffda548/tensorboard/plugins/histogram/summary_v2.py#L142-L146